packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.spec.js

```diff
@@ -32,12 +31,15 @@ const failSubject = Factory.build('subject', {
 
 describe('Component > LightCurveViewerContainer', function () {
   it('should render without crashing', function () {
-    wrapper = shallow(<LightCurveViewerContainer.wrappedComponent />)
+    wrapper = shallow(<LightCurveViewerContainer onKeyDown={() => {}} setOnPan={() => {}} setOnZoom={() => {}} />)
```

Comment:
The tests run without these callback props so I'm not sure that they're needed.

Reply:
The tests run, but then we get the props warnings about them not being defined since I now set them as required, which is annoying.

Reply:
Yeah, and the component renders happily without them so I'm not seeing why they're required in order for it to run. My interpretation would be that the component should error if required props aren't present when it tries to render.

Reply:
I'd like to refactor it to error if a zoom function isn't set. That's a next step I think.